scripts/source-maintenance.sh

```diff
@@ -35,6 +35,14 @@ while [ $# -ge 1 ]; do
         KeepGoingDirective=$1
         shift
         ;;
+    --skip-astyle-version-check)
+        SkipAstyleVersionCheck=yes
+        shift
+        ;;
+    --verbose|-v)
+        Verbose=yes
+        shift
+        ;;
```

Comment:
I recommend removing these out-of-scope changes to limit the discussion to in-scope items.

Reply:
They were helpful for testing, but sure